jobs/build/drop_advisories/Jenkinsfile

```diff
@@ -43,12 +49,14 @@ node {
         res = commonlib.shell(
             returnAll: true,
             script: """
+              ${buildlib.ELLIOTT_BIN} repair-bugs --advisory ${adv} --all --comment ${comment} --close-placeholder --from RELEASE_PENDING --to VERIFIED
```

Comment:
`${comment}` is likely to contain whitespaces. In that case, this command will be broken.